test/NuGet.Clients.FuncTests/NuGet.MSSigning.Extensions.FuncTest/Commands/MSSignCommandTests.cs

```diff
@@ -137,7 +139,7 @@ public void GetAuthorSignRequest_InvalidFingerprint()
                     CertificateFile = test.CertificatePath,
                     CSPName = test.CertificateCSPName,
                     KeyContainer = test.CertificateKeyContainer,
-                    CertificateFingerprint = "invalid-fingerprint",
```

Comment:
An invalid fingerprint is still a good negative test case.  Instead of replacing this value, I suggest converting this test from a `Fact` into a `Theory` and test both cases, like you did [here](https://github.com/NuGet/NuGet.Client/pull/5932/files#diff-ebac56a1632255034931687d8b4bec09c58e9dfe12b6482166672639dce68ff6R181-R184).

Reply:
In this test case we are validating that if an invalid hexadecimal hash has been passed to the certificate fingerprint, then then the command will fail with `Can't find specified certificate.` exception.

Once this PR has been merged, the command will throw an exception with NU3034 error code if non-hexadecimal string is passed as fingerprint. 

As per your suggestion, I changed the test to pass invalid SHA1 and SHA256 hexadecimal hashes to ensure the certificate lookup fails with the expected exception.